src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -30,6 +30,11 @@
 from .util import random_split_by_session as _random_split_by_session
 from .util import _MIN_NUM_SESSIONS_FOR_SPLIT
 
+import os
```

Comment:
Always import using _os

Reply:
That good general practice and it's not going to hurt here, but I don't think it's needed. The filename for this file starts with an underscore so we're already in a hidden scope.